### PR TITLE
[V2 API] Update observation series API to V2

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
nice!

File: server/integration_tests/test_data/sdg/query_2/chart_config.json [hunk 1]
```diff
@@ -386,35 +386,35 @@
                     "type": "LINE"
                   },
                   {
-                    "placeDcidOverride": "country/CIV",
+                    "placeDcidOverride": "country/SHN",
```
[the remark this answers is not in the document]
Comment (pradh) [reply server/integration_tests/test_data/sdg/query_2/chart_config.json:389]: This list of places is computed from the results of series_core in rank_places_by_series_growth(): https://github.com/datacommonsorg/website/blob/30aa9b9d678a5f66d9de86f8902e4d1ff369073f/server/lib/nl/utils.py#L194-L196

Does the implementation somehow change the order of places?

[the remark this answers is not in the document]
Comment (shifucun) [reply server/integration_tests/test_data/sdg/query_2/chart_config.json]: This is due to unstable sort when growth params are all the same for different places. Use place name for sorting as well to make it stable.

**Review by pradh — APPROVED** [review]
Thanks for the sort fix!

**Pull request merged**